python_src/src/lamp_py/performance_manager/l0_gtfs_rt_events.py

```diff
@@ -255,13 +244,16 @@ def upload_to_database(
     events["vp_move_timestamp"] = events["vp_move_timestamp"].astype("Int64")
     events["vp_stop_timestamp"] = events["vp_stop_timestamp"].astype("Int64")
     events["tu_stop_timestamp"] = events["tu_stop_timestamp"].astype("Int64")
+    events["vp_stop_db"] = events["vp_stop_db"].astype("Int64")
+    events["vp_move_db"] = events["vp_move_db"].astype("Int64")
     events["pk_id"] = events["pk_id"].astype("Int64")
+    events = events.fillna(numpy.nan).replace([numpy.nan], [None])
```

Comment:
Dataframe value cleanup to ensure `do_update` and `do_insert` masks behave properly

Reply:
what does the _db stand for?

Reply:
basically means "from database", these columns and the move and stop timestamps pulled from the database for comparison purposes, open to different nomenclature